data_diff/dbt.py

```diff
@@ -0,0 +1,313 @@
+import os
+from typing import List
+import requests
+import json
+import yaml
+import rich
+import logging
+
+from dataclasses import dataclass
+from packaging.version import parse as parse_version
+from pathlib import Path
+from dbt_artifacts_parser.parser import parse_run_results, parse_manifest
+from . import connect_to_table, diff_tables, Algorithm
+from dbt.config.renderer import ProfileRenderer
+
+CONST_RUN_RESULTS_PATH = "/target/run_results.json"
+CONST_MANIFEST_PATH = "/target/manifest.json"
+CONST_PROJECT_FILE = "/dbt_project.yml"
+CONST_PROFILES_FILE = "/profiles.yml"
+CONST_DBT_V_ABOVE = "1.0.0"
+CONST_DBT_V_BELOW = "1.4.0"
```

Comment:
**nit:** There is no need to explicitly mention `CONST_…` for constants — it is already clear from the conventional name (all-caps). Just `RUN_RESULTS_PATH` would be fine.

Reply:
fixed